UnityProject/Assets/Scripts/Player/PlayerSync.Server.cs

```diff
@@ -570,10 +570,13 @@ private PlayerState NextStateServer(PlayerState state, PlayerAction action)
 	///Revert client push prediction straight ahead if it's wrong
 	[Command]
 	private void CmdValidatePush( GameObject pushable ) {
-		var pushPull = pushable.GetComponent<PushPull>();
-		if ( Validations.CanInteract(playerScript, NetworkSide.Server) || pushPull && !playerScript.IsInReach(pushPull.registerTile, true) ) {
-			questionablePushables.Add( pushPull );
-			Logger.LogWarningFormat( "Added questionable {0}", Category.PushPull, pushPull );
+		if (pushable && pushable.TryGetComponent(out PushPull pushPull))
+		{
+			if (Validations.CanInteract(playerScript, NetworkSide.Server) || pushPull && !playerScript.IsInReach(pushPull.registerTile, true))
+			{
+				questionablePushables.Add(pushPull);
+				Logger.LogWarningFormat("Added questionable {0}", Category.PushPull, pushPull);
+			}
 		}
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
PlayerSync.CallCmdValidatePush (UnityEngine.GameObject pushable) (at /github/workspace/UnityProject/Assets/Scripts/Player/PlayerSync.Server.cs:573)
PlayerSync.InvokeCmdCmdValidatePush (Mirror.NetworkBehaviour obj, Mirror.NetworkReader reader) (at <23204bcecd804e499e9fb0ecab42bc98>:0)
Mirror.NetworkBehaviour.InvokeHandlerDelegate (System.Int32 cmdHash, Mirror.MirrorInvokeType invokeType, Mirror.NetworkReader reader) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkBehaviour.cs:449)
Mirror.NetworkIdentity.HandleRemoteCall (System.Int32 componentIndex, System.Int32 functionHash, Mirror.MirrorInvokeType invokeType, Mirror.NetworkReader reader) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkIdentity.cs:928)
Mirror.NetworkIdentity.HandleCommand (System.Int32 componentIndex, System.Int32 cmdHash, Mirror.NetworkReader reader) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkIdentity.cs:948)
Mirror.NetworkServer.OnCommandMessage (Mirror.NetworkConnection conn, Mirror.CommandMessage msg) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:901)
Mirror.MessagePacker+<>c__DisplayClass6_0`2[T,C].<MessageHandler>b__0 (Mirror.NetworkConnection conn, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146)
Mirror.NetworkConnection.InvokeHandler (System.Int32 msgType, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219)
Mirror.NetworkConnection.TransportReceive (System.ArraySegment`1[T] buffer, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:272)
Mirror.NetworkServer.OnDataReceived (System.Int32 connectionId, System.ArraySegment`1[T] data, System.Int32 channelId) (at /github/workspac